hpcbench/benchmark/osu.py

```diff
@@ -257,7 +259,7 @@ def __init__(self):
         super(OSU, self).__init__(
             attributes=dict(
                 categories=OSU.DEFAULT_CATEGORIES,
-                arguments=OSU.DEFAULT_ARGUMENTS,
+                options=OSU.DEFAULT_ARGUMENTS,
```

Comment:
Please rename `DEFAULT_ARGUMENTS` constant to `DEFAULT_OPTIONS`

Reply:
Done!